azure-steps-windows.yml

```diff
@@ -14,7 +14,8 @@ steps:
   displayName: 'Install dependencies; some are optional to avoid test skips'
 
 - powershell: |
-    choco install -y rtools
+    choco install -y rtools --noprogress --force --version=4.0.0.20220206
+    $env:RTOOLS40_HOME="c:\\rtools40"
     refreshenv
```

Comment:
It is needed because changes to $env: are local to each invocation of powertools.

Reply:
Ah, right... but neither seems to have worked.  `RTOOLS40_HOME` isn't defined, it now prints the hole list successfully (sorry, I guess I can't link to the previous version, but only `RTOOLS43_HOME` *is* defined and the paths are added, but are just missing the `RTOOLS40_HOME` part consistent with it not being defined.

So, it seems `RTOOLS40_HOME` isn't being inherited by later blocks if we add it like this.

Reply:
:shrug: want to try to just add the `$env:RTOOLS40_HOME = "C:\rtools40\"` line everywhere where `RTOOLS40_HOME` is otherwise used (openblas step and the test step I guess)?

Maybe that works, and who knows, maybe @andyfaff has a tip on how to make it less of a mess...